examples/keyCopy.c

```diff
@@ -12,65 +12,37 @@
 
 Key * copy;
 
+//! [Duplicate Key]
+void x (Key * orig)
+{
+	copy = keyCopy (keyNew ("/", KEY_END), orig, KEY_CP_ALL);
```

Comment:
I thought we want to use `keyCopy( NULL` in such situations?

Reply:
AFAIK the latest decision (https://github.com/ElektraInitiative/libelektra/pull/3606#issuecomment-754835903) was to add the alias `keyDup`. However, `keyCopy (keyNew ("/", KEY_END), orig, KEY_CP_ALL)` is the canonical way to duplicate `orig` in any case.

Even, if we allow `keyCopy (NULL, k, flags)`, this would just be defined as an (internal) alias for `keyCopy (keyNew ("/", KEY_END), orig, KEY_CP_ALL)`. That's why I mention this first in the docs, and then explain that there is an alias to keep it short.